packages/client/src/index.ts

```diff
@@ -25,6 +25,8 @@ export * as Objects from "./object/index.js";
 export type { OsdkObject } from "./OsdkObject.js";
 export type { Osdk } from "./OsdkObjectFrom.js";
 
+export type { WhereClause } from "./query/WhereClause.js";
```

Comment:
I think you may need to rebase because I _think_ you already exported this in a PR last week right?